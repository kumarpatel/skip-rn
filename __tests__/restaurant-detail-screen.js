import React from "react";
import renderer from "react-test-renderer";
import RestaurantDetailScreen from "../screens/restaurant-detail-screen";
import mockData from "./mockSearchResults.json";

it("Snapshot - RestaurantDetailScreen", () => {
  const tree = renderer
    .create(
      <RestaurantDetailScreen
        route={{
          params: {
            restaurant: mockData.Restaurants[0],
          },
        }}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
