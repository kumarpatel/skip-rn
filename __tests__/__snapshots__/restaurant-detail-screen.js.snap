// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot - RestaurantDetailScreen 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "backgroundColor": "#fff",
    }
  }
>
  <View>
    <View
      accessible={true}
      focusable={true}
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
    >
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "backgroundColor": "#fff",
            "borderColor": "grey",
            "borderRadius": 5,
            "borderWidth": 2,
            "flex": 1,
            "justifyContent": "center",
            "marginLeft": 5,
            "marginRight": 5,
            "marginTop": 10,
            "padding": 10,
          }
        }
      >
        <Image
          source={
            Object {
              "uri": "http://d30v2pzvrfyzpo.cloudfront.net/uk/images/restaurants/80086.gif",
            }
          }
          style={
            Object {
              "height": 70,
              "width": 70,
            }
          }
        />
        <View
          style={
            Object {
              "display": "flex",
              "flexDirection": "column",
            }
          }
        >
          <View
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <Text>
              Name: 
              La Venice Pizzeria (Wood Fired)
            </Text>
          </View>
          <View
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <Text>
              Rating: 
              5.08
            </Text>
          </View>
          <View
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          >
            <Text>
              Cuisine Types:
               
              Italian | Pizza
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#fff",
        }
      }
    >
      <Text>
        Address:
      </Text>
      <Text>
        242 Old Kent Road
      </Text>
      <Text>
        London SE1 5UB
      </Text>
      <Text>
        
      </Text>
      <Text>
        Opening Time: 
        7:00:00 AM
      </Text>
      <Text>
        Delivery Time:
         
        8:00:00 PM
      </Text>
      <Text>
        
      </Text>
      <Text>
        Deals:
      </Text>
      <Text>
        No deals available today
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
